Source/santa/SNTNotificationManager.m

```diff
@@ -112,12 +114,49 @@ - (void)queueMessage:(SNTMessageWindowController *)pendingMsg {
 
   pendingMsg.delegate = self;
   [self.pendingNotifications addObject:pendingMsg];
+  [self postDistributedNotification:pendingMsg];
 
   if (!self.currentWindowController) {
     [self showQueuedWindow];
   }
 }
 
+- (void)postDistributedNotification:(SNTMessageWindowController *)pendingMsg {
```

Comment:
nit: Add a comment here describing what a distributed notification is.

Reply:
Done